packages/editor/src/pages/Auth/index.tsx

```diff
@@ -57,20 +59,23 @@ class AuthPage extends React.Component<IProps, IState> {
     this.params = queryString.parse(queryString.extract(window.location.href));
 
     let base64Key: string | undefined;
-    if (typeof this.params.key === 'string' && this.params.key.trim().length > 0) {
+    if (this.params.key && this.params.key.trim().length > 0) {
       base64Key = this.params.key;
 
       // If landed on the page and have a "key" query parameter, the window
-      // must be re-used of a new auth flow.  So just in case,
+      // might be re-used for a new auth flow.  So just in case,
       // clear the session storage, and then store the key parameter
-      sessionStorage.clear();
-      sessionStorage.setItem(SESSION_STORAGE_AUTH_KEY_PARAMETER, base64Key);
+      for (const keyName in AUTH_PAGE_SESSION_STORAGE_KEYS) {
+        sessionStorage.removeItem(AUTH_PAGE_SESSION_STORAGE_KEYS[keyName]);
+      }
+
+      sessionStorage.setItem(AUTH_PAGE_SESSION_STORAGE_KEYS.auth_key, base64Key);
     } else {
-      base64Key = sessionStorage.getItem(SESSION_STORAGE_AUTH_KEY_PARAMETER); // or undefined
+      base64Key = sessionStorage.getItem(AUTH_PAGE_SESSION_STORAGE_KEYS.auth_key); // or undefined
```

Comment:
intentional comment?

Reply:
It was, but I agree that it looks like it could have been an accidental one.  Let me rework it.